docs/ADOPTION.md

```diff
@@ -0,0 +1,91 @@
+# Adopting TFCI
+
+If you have any questions or need clarification feel free to open an [issue](https://github.com/hashicorp/tfc-workflows-tooling/issues).
+
+## Setup
+
+Tfci currently supports the following CI/CD platforms:
+* [GitHub Actions](https://docs.github.com/en/actions)
+* [GitLab Pipelines](https://docs.gitlab.com/ee/ci/pipelines/)
+
+Tfci can be instrumented for other platforms with the use of the [published Docker Container](https://hub.docker.com/r/hashicorp/tfci).
+
+### Docker
+
+Tfci generates an Docker container artifact that is available from the Docker public registry, `docker://hashicorp/tfci:{VERSION}`.
+
+Leveraging the publicly distributed Docker image is the recommended approach for running Tfci on CI/CD platforms for non-human interactions.
+
+View our [Usage documentation](./USAGE.md) to learn more our available commands, arguments, and configuration.
+
+### Building a binary from source code
+
+If Docker is not available, view our [Usage documentation](./USAGE.md#generating-a-binary-from-source) to learn more.
+
+### How GitHub Actions uses tfci
+
+[View all](https://github.com/hashicorp/tfc-workflows-github/tree/main/actions) available GitHub Actions that are built on top of tfci.
+
+### [How GitLab Pipelines uses tfci](https://github.com/hashicorp/tfc-workflows-gitlab)
+
+View the GitLab [Base-Template](https://github.com/hashicorp/tfc-workflows-gitlab/blob/main/Base.gitlab-ci.yml)
+
+## Workflow
+
+### [Terraform Cloud CLI](https://developer.hashicorp.com/terraform/cloud-docs/run/cli) vs. [Terraform Cloud API](https://developer.hashicorp.com/terraform/cloud-docs/run/api)
+
+| Workflow   |    Terraform CLI (Cloud)            |  TFCI/Terraform Cloud API                      |
+|------------|-------------------------------------|------------------------------------------------|
+| Plan       |  `terraform plan`                   |  commands: `upload`, `run create`              |
+| Apply      |  `terraform apply -auto-approve`    |  commands: `upload`,  `run create`, `run apply`|
+
+#### Terraform Plan
+
+Terraform Cloud CLI can execute a new plan with one command that will upload Terraform configuration and execute a new run in Terraform Cloud.
+
+With tfci and Terraform Cloud API driven runs, these actions are broken up into multiple parts:
```

Comment:
Q: Should tfci be capitalized throughout this doc?

Reply:
Yeah, good eye!